app/views/katello/api/v2/host_collections/_host_collection.json.rabl

```diff
@@ -1,4 +1,5 @@
 attribute :pulp_id, :name, :organization_id, :max_hosts, :description, :total_hosts, :unlimited_hosts
+attributes :host_ids
```

Comment:
I should say and change it to be like products or repositories in that  the index rabl doesn't use show.json.rabl